wdl2cwl/main.py

```diff
@@ -654,6 +654,11 @@ def get_expr_apply(self, wdl_apply_expr: WDL.Expr.Apply) -> str:
             array_items = [str(self.get_expr(item)) for item in array_obj.items]  # type: ignore
             items_str = ", ".join(array_items)
             return f"[{items_str}].find(element => element !== null) "
+        elif function_name == "select_all":
+            array_obj = cast(WDL.Expr.Array, arguments[0])
+            array_items = [str(self.get_expr(item)) for item in array_obj.items]
+            items_str = ", ".join(array_items)
+            return f"[{items_str}].filter(element => element !== null) "
```

Comment:
Basically copied the `select_first`, but instead of JS's `.find`, used `.filter`.